tests/openvino/test_modeling.py

```diff
@@ -2220,6 +2232,15 @@ def test_compare_to_transformers(self, model_arch):
         set_seed(SEED)
         ov_outputs = ov_model.generate(**inputs, generation_config=gen_config)
         set_seed(SEED)
+
+        # TODO: add back once https://huggingface.co/katuni4ka/tiny-random-maira2/discussions/1 merged as current mdoeling incompatible with transformers >= v4.49
+        if model_arch in {"maira2"} and is_transformers_version(">=", "4.49"):
```

Comment:
fixed in https://huggingface.co/katuni4ka/tiny-random-maira2/discussions/1

Reply:
https://huggingface.co/katuni4ka/tiny-random-maira2/discussions/1 merged, removing in https://github.com/huggingface/optimum-intel/pull/1172/commits/3c77176b72fbcc13a8068717145214829fae4b11